configure.ac

```diff
@@ -372,23 +372,6 @@ case $CSPREFIX in
   prefix) CSPREFIX=$prefix;;
 esac
 
-AC_MSG_NOTICE(Sanitizing INITDIR: ${INITDIR})
-case $INITDIR in
-  prefix) INITDIR=$prefix;;
-  "")
-    AC_MSG_CHECKING(which init (rc) directory to use)
-      for initdir in /etc/init.d /etc/rc.d/init.d /sbin/init.d	\
-	   /usr/local/etc/rc.d /etc/rc.d
```

Comment:
This isn't "overly smart" -- it's a simple list of common locations, and avoids unnecessary extra work for our users to figure out local details, which is the point of the configure script. Someone building pacemaker (possibly in a heterogeneous environment) shoudn't have to be aware of this option or find an appropriate value, for the common case of running "./configure".